### ISSUE TITLE
How to fallback to default path completions

### ISSUE THREAD

**Issue by uroybd** [nushell/nushell#14059]
### Question

I'm using `zoxide` instead of `cd` with this completion setup: https://www.nushell.sh/cookbook/external_completers.html#zoxide-completer

While it works well for directories already recorded in `zoxide`, it doesn't provide the natural path completion like cd.

How to setup the completion engine to show both zoxide query and paths?

**Comment by fdncred** [comment]
I get path completions with `z` to regular directories that aren't in zoxide's database. I don't use the completer you listed though. Maybe that's the difference?

**Comment by uroybd** [comment]
> I get path completions with `z` to regular directories that aren't in zoxide's database. I don't use the completer you listed though. Maybe that's the difference?

Yes. It does path completion without the completer I'm using. But then, no zoxide query there.

**Comment by fdncred** [comment]
I use `zi` for `zoxide query`. I'm pretty sure zoxide installs that by default.

**Comment by uroybd** [comment]
> I use `zi` for `zoxide query`. I'm pretty sure zoxide installs that by default.

But, isn't there a way to get the completion for `cd` by invoking a function or something?

**Comment by fdncred** [comment]
maybe shadowing the cd command with an alias like `alias cd = z`, `alias cdi = zi`. I haven't tried.

**Comment by WindSoilder** [comment]
Closing the issue because the question is answered.  Thank you @songpola
